apps/vr-tests-react-components/.storybook/preview.js

```diff
@@ -53,7 +53,8 @@ setAddon({
   },
 });
 
-export const parameters = { layout: 'none' };
+/** @type {import("@fluentui/react-storybook-addon").FluentParameters} */
+export const parameters = { layout: 'none', mode: 'vr-test' };
```

Comment:
we should improve our types by this change in our addon 

<img width="1918" alt="image" src="https://user-images.githubusercontent.com/1223799/199315894-5b5fd086-17f1-4572-93a3-c1aade93061b.png">

with that we will get proper DX/type-checks for `layout` and other params that come with core


**with the change:**

<img width="784" alt="image" src="https://user-images.githubusercontent.com/1223799/199316151-ab4950de-1028-40aa-a5c1-cdcddfa52620.png">